ccxt.js

```diff
@@ -156,6 +156,7 @@ const exchanges = {
     'quoinex':                 require ('./js/quoinex.js'),
     'rightbtc':                require ('./js/rightbtc.js'),
     'southxchange':            require ('./js/southxchange.js'),
+    'stocksexchange':          require ('./js/stocksexchange.js'),
```

Comment:
You don't have to commit this file, as this particular line will be added upon building automatically ) JFYI